src/serviceFacades/apps.js

```diff
@@ -221,6 +366,13 @@ export {
     searchAppsInfiniteQuery,
     getAppDoc,
     saveAppDoc,
+    adminPatchApp,
+    adminAddAppDoc,
+    adminUpdateAppDoc,
+    adminGetAppAVUs,
+    addAVUToApp,
+    adminSetAppAVUs,
```

Comment:
```suggestion
    adminGetAppAVUs,
```
So I think these are the functions that can be removed from the export.